src/dynarec/arm64/dynarec_arm64_f0.c

```diff
@@ -247,7 +247,7 @@ uintptr_t dynarec64_F0(dynarec_arm_t* dyn, uintptr_t addr, uintptr_t ip, int nin
                                 CBNZx_MARKLOCK(x4);
                                 // done
                                 MARK;
-                                UFLAG_IF {emit_cmp32(dyn, ninst, rex, x6, x2, x3, x4, x5);}
+                                UFLAG_IF {emit_cmp8(dyn, ninst, rex, x6, x2, x3, x4, x5);}
```

Comment:
remove `rex` argument, it's not relevent for 8bits operations

Reply:
Oh, indeed.